document-generator-core/src/main/java/uk/gov/companieshouse/document/generator/core/models/DocumentGeneratorRequest.java

```diff
@@ -0,0 +1,80 @@
+package uk.gov.companieshouse.document.generator.core.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.validation.constraints.NotNull;
+
+public class DocumentGeneratorRequest {
```

Comment:
i would add the `toString` method here as its good practice to include them in POJOs

Reply:
done

Reply:
We could use GSON or Jackson in the `toString` method which would remove any risk of a variable name mistype. Not needed for this PR, just food for thought.